Sources/Plasma/PubUtilLib/plScene/plSceneNode.cpp

```diff
@@ -160,9 +160,9 @@ void plSceneNode::CollectForRender(plPipeline* pipe, hsTArray<plDrawVisList>& le
     {
         if( pipe->PreRender(fDrawPool[idx], visSpans, visMgr) )
         {
-            plDrawVisList* drawVis = levList.Push();
-            drawVis->fDrawable = fDrawPool[idx];
-            drawVis->fVisList.swap(visSpans);
+            plDrawVisList& drawVis = levList.emplace_back();
+            drawVis.fDrawable = fDrawPool[idx];
+            drawVis.fVisList.swap(visSpans);
```

Comment:
This could be another construct in-place, but it would defeat the little visSpan "optimization" Cyan has going.